test_pep257.py

```diff
@@ -76,7 +76,8 @@ def test_ignore_list():
     no blank line after one-liner is bad. Also this - \"\"\"
     return foo
     """
-    expected_error_codes = set(('D100', 'D400', 'D401', 'D205', 'D209'))
+    expected_error_codes = set((
+        'D100', 'D400', 'D401', 'D205', 'D209', 'D210'))
```

Comment:
Break the line as part of the list:

    expected_error_codes = set(('D100', 'D400', 'D401', 'D205', 'D209',
                                'D210'))